src/components/FormButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+type FormButtonProps = {
+  disabled: boolean;
+  type: "button" | "submit" | "reset";
+  label: string;
+};
+
+const FormButton: React.FC<FormButtonProps> = ({ label, disabled, type }) => {
```

Comment:
I'm not really sure what this component does, as it is just rendering a "plain" button. Why not use the button directly instead?

Reply:
You're right, it's not necessary here. I think I mostly got caught up thinking about making the button component resuable and able to take additional props (like loading and error states to change the render); none of which I did here.